polyfill/test/regex.mjs

```diff
@@ -33,7 +33,7 @@ describe('fromString regex', ()=>{
       test(`${dateTimeString}:30.123456789${zoneString}`, components);
     }
     // Time separators
-    ['T', ' '].forEach((timeSep) =>
+    ['T', 't', ' '].forEach((timeSep) =>
```

Comment:
I'd say based on https://github.com/tc39/proposal-temporal/issues/198#issuecomment-552404254 and https://github.com/tc39/proposal-temporal/issues/198#issuecomment-553225994 we should only accept a space and not `\s+`, since a single space is mentioned as an alternate character in RFC 3339. However, maybe multiple whitespaces are commonly output and that was the rationale for adding it ... @pipobscure?

Reply:
The rationale was actually a habit of mine that I always use `\s+` when looking for a space 😄  so nothing as well thought out as that.

So yes let’s replace that with just matching a single space character.